db/insertDummyData.js

```diff
@@ -1,21 +1,12 @@
 const db = require('./db.js');
 const data = require('./dummyData.js');
-const pgp = require('pg-promise')();
 const path = require('path');
-const pg = require('pg');
 
-// This is here because env LOCAL DB not working
-// const config = {
-//   host: 'localhost',
-//   port: 5432,
-//   database: 'gecko'
-// };
-
-// const db = pgp(config);
+var QueryFile = db.$config.pgp.QueryFile;
```

Comment:
Not sure on the pros and cons of one vs the other.  Vitaly just suggested it so I took his word